### ISSUE TITLE
support for solo play

### ISSUE THREAD

**Issue by TrekkieByDay** [maxpaulus43/owlbear-shadowdark-character-sheet#48]
Currently, the "GM role" in OBR 2.0 is not allowed to load their own character sheets. Would be really great if they could do that to support playing Solo Shadowdark. Arcane Library has released "Solodark" rules, and there is a lot of interest. As it stands now, to get it to work properly, I have to log in from a secondary OBR account to load the sheet, which then can be loaded (but not saved) by the GM. This greatly diminishes the viability for this extension in support of solo play.

**Comment by maxpaulus43** [comment]
hmm good point. i'll try to find some time this week to add that

**Comment by thomashorta** [comment]
@TrekkieByDay I also wanted to use OBR with Solodark to get more familiar with Shadowdark and OBR. I discovered this extension yesterday and really liked it but I got stuck in the same issue of not being able to have GM Character sheets.

Today I had some free time on my hands and pushed a PR adding support for it though (#53) so hopefully @maxpaulus43 is able to pull the change soon and we get that in a next release!
